src/styles/utils/variables.scss

```diff
@@ -0,0 +1,17 @@
+$main-accent-color: #060b35;
+$secondary-color: #616070;
+$blue-accent-color: #00acdc;
+$card-border-color: #f3f3f3;
+$white-color: #fff;
+$main-font-size: 12px;
+$secondary-font-size: 10px;
+$accent-font-size: 16px;
+$button-font-size: 14px;
```

Comment:
too many variables that are used 1 time.
You don't need these variables for fonts(except for the colors of course)).
In the future, I will show how to correctly create a file for fonts(Don't worry about it yet)